server/src/main/java/org/opensearch/index/store/remote/filecache/FileCache.java

```diff
@@ -153,4 +180,23 @@ public void restoreFromDirectory(List<Path> fileCacheDataPaths) {
                 }
             });
     }
+
+    /**
+     * Returns the current {@link FileCacheStats}
+     */
+    public FileCacheStats fileCacheStats() {
```

Comment:
This seems like a much better home for this method than NodeEnvironment, but we now do have both `FileCache::stats` and `FileCache::fileCacheStats` methods here. Perhaps something to revisit in a subsequent PR.

Reply:
Yep. I realized that and thought of the same. We can clean it up with another PR, but this looked like the right place for it to be in.